android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/EventSource.java

```diff
@@ -570,6 +570,7 @@ public static final class Builder {
     private int maxEventTasksInFlight = 0;
     private boolean streamEventData;
     private Set<String> expectFields = null;
+    private ExecutorService streamExecutor;
```

Comment:
do we need an import statement here?

Reply:
I think it's imported from line 17🤔 `import java.util.concurrent.ExecutorService;`